misc/inmanta.cfg

```diff
@@ -107,3 +107,6 @@ port = 8888
 enabled=true
 # The path where the dashboard is installed
 path=/opt/inmanta/dashboard
+
+[web-console]
```

Comment:
This is not strictly required (enabled by default) but I believe it helps to follow the installation instructions that this section already exists.